webapp/src/translations/languages/en.json

```diff
@@ -964,9 +964,9 @@
     "walletUnlockMasternodeMessage": "You are currently running a masternode on a locked wallet. Please unlock your wallet with your passphrase to continue minting.",
     "lostPassphraseMessage": "I lost my passphrase",
     "backupWalletWarningModelHeader": "Wallet backup warning",
-    "backupNow": "Backup now",
+    "backupNow": "Back up now",
```

Comment:
Used as a noun, keep it as "Backup now"

Reply:
I disagree. "Back up" is a verb here and really cannot be used as a noun. I suggest the wording "create backup" instead.

Reply:
Over here, it's also an action button. t's also used with consistency with "Backup wallet" in other parts of the app. I'm rejecting this.